docs/getting_started.md

```diff
@@ -1,9 +1,33 @@
 Getting Started
 ================
-1. Run the `setup.sh` script to find & replace placeholders with information about your project
-2. Fill out `README.md` with details about your project
-3. Edit documentation configuration in `conf.py`
-    - For the full list of available options, see [the official documentation][Sphinx-docs].
-4. Rewrite this page with instructions for how to get started with your project
+
+### Setup
+1. Ensure batteries are charged
+2. Connect flight computers via USB cable
+3. Start AltOS
+4. Download and delete all on-board flight logs
```

Comment:
Might want to clarify what is being deleted. The ambiguity here is that someone could download the flight logs and then delete the flight logs they downloaded instead of downloading the logs and then clearing the logs from the flight controller.

Reply:
on-board denotes you are removing the flight logs from the flight computer